modules/tinymce/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Added
+- Updated the image dialog to show a grid with two columns only if `hasImageCaption` is true, `image` plugin #TINY-6400
```

Comment:
I think this should maybe in the `Fixed` section instead of `Added`?
Possible alternative suggestion:
```suggestion
- Updated the image dialog to display the class list dropdown as full-width if the caption checkbox is not present #TINY-6400
```

Reply:
updated